odoo/addons/base/rng/common.rng

```diff
@@ -278,7 +278,6 @@
             <rng:optional><rng:attribute name="decoration-primary"/></rng:optional>
             <rng:optional><rng:attribute name="decoration-success"/></rng:optional>
             <rng:optional><rng:attribute name="decoration-warning"/></rng:optional>
-            <rng:optional><rng:attribute name="kanban_view_ref" /></rng:optional>
```

Comment:
I'm not sure if we should remove this in stable, I think if we remove it, view with the attribute might not be editable until the user remove the kanban_view_ref, but he might not be technical enough to do it (and he might use studio which make it even more technical to do it)

in master I don't see an issue to do it